test/evp_extra_test.c

```diff
@@ -6855,6 +6855,71 @@ static int test_evp_cipher_pipeline(void)
     return testresult;
 }
 
+/*
+ * Test that AEAD decrypt final without setting a tag fails and pushes an EVP error
+ */
+static int test_evp_aead_missing_tag_error(void)
+{
+    int ret = 0;
+    EVP_CIPHER_CTX *enc = NULL, *dec = NULL;
+    const EVP_CIPHER *ciph = NULL;
+    unsigned char key[16] = {0};
+    unsigned char iv[12] = {0};
+    unsigned char pt[32] = { 'H','e','l','l','o',' ','A','E','A','D' };
+    unsigned char ct[64];
+    int outl = 0, tmplen = 0;
+    unsigned long err = 0;
+
+    enc = EVP_CIPHER_CTX_new();
+    dec = EVP_CIPHER_CTX_new();
+    if (!TEST_ptr(enc) || !TEST_ptr(dec))
+        goto end;
+
+    ciph = EVP_aes_128_gcm();
```

Comment:
Change this to use the following
ciph =  EVP_CIPHER_fetch(testctx, "AES-128-GCM", testpropq);

This test file uses a library context.
If you pass EVP_aes_128_gcm() into the EncryptInit() then internally it does a fetch that uses EVP_CIPHER_fetch(NULL, alg, NULL)..

Reply:
Ah thank you. Hopefully that was the cause of the CI test failures I could see, which failed because of a non-test context.